portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,25 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: 'Source Serif Pro';
```

Comment:
Might be worth alphabetizing the sections, so it is a little cleaner when adding more.

Reply:
I'll put the change in